src/i18n.ts

```diff
@@ -34,7 +34,8 @@ const resources = {
       Cancel: 'Cancel',
       'Wallet address': 'Wallet address',
       Network: 'Network',
-      'Don’t show again': 'Don’t show again'
+      'Don’t show again': 'Don’t show again',
```

Comment:
This apostrophe _could_ cause encoding issues with some browsers especially on windows. For the key, you could leave it out, and the value either escape it with \'\ or use the encoded version of `&#39;`

i.e.: `Don&#39;t show again`